test/torch/pointers/test_pointer_tensor.py

```diff
@@ -17,12 +17,47 @@ def test_create_pointer(workers):
 
 
 def test_send_default_garbage_collector_true(workers):
-    """Pointer tensor should be garbage collected by default."""
+    """ emote tensor should be garbage collected by default on deletion of the Pointer tensor pointing to remote tensor"""
```

Comment:
emote -> remote. Also could you split this up into 2 lines (its a bit wide on my screen)